.changeset/lovely-poems-remember.md

```diff
@@ -0,0 +1,20 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+- Added data attribute 'data-iui-overflow' - when true it adds styling for overflow tabs
+- Added property 'overflowOptions' - contains `useOverflow`, which when true enables tabs to scroll if there's overflow
+
+```typescript
+const [index, setIndex] = React.useState(0);
```

Comment:
Keep this example as simple as possible. I don't think users need useState or return statements.